client/scripts/start.sh

```diff
@@ -15,7 +15,7 @@ realpath() {
 FILE_PATH=$(dirname $(realpath "$0"))
 source $FILE_PATH/../../.env.test
 
-nohup yarn serve --port 58080 >server.out &
+FLASK_SERVE_PORT=$FLASK_SERVE_PORT nohup yarn serve --port 58080 >server.out &
 echo $! >server.pid
```

Comment:
Why is it required to do this? Since we're already sourcing `.env.test` isn't this essentially a no op?

Reply:
It shouldn't be, I'm just troubleshooting. Having some issues with sourcing the .env.test on the ubuntu runner. I'm spinning up a local ubuntu instance here to test